WalletWasabi.Fluent/Styles/Themes/BaseLight.axaml

```diff
@@ -18,9 +22,9 @@
 
     <SolidColorBrush x:Key="ActionButtonForegroundColor" Color="White" />
     <SolidColorBrush x:Key="ActionButtonBackgroundColor" Color="{DynamicResource SystemAccentColor}" />
-    <SolidColorBrush x:Key="ActionButtonPressedBackgroundColor" Color="#608522" />
-    <SolidColorBrush x:Key="ActionButtonPointerOverBackgroundColor" Color="#719E26" />
-    <SolidColorBrush x:Key="ActionButtonDisabledBackgroundColor" Color="#80A3A3A3" />
+    <SolidColorBrush x:Key="ActionButtonPressedBackgroundColor" Color="{DynamicResource SystemAccentColorDark1}" />
+    <SolidColorBrush x:Key="ActionButtonPointerOverBackgroundColor" Color="{DynamicResource SystemAccentColorLight1}" />
+    <SolidColorBrush x:Key="ActionButtonDisabledBackgroundColor" Color="{DynamicResource SystemControlDisabledBaseLowBrush}" />
```

Comment:
These are all the same in light and dark themes. Can you move them into the `Base.axaml`?